boards/nordic/nrf54l15dk/Kconfig.defconfig

```diff
@@ -12,6 +12,20 @@ config ROM_START_OFFSET
 
 endif # BOARD_NRF54L15DK_NRF54L15_CPUAPP
 
+# What MCUboot requires is here
+if MCUBOOT
+
+# Within sdk-nrf NRF security is default
+config NRF_SECURITY
+	default y
+	select NRF_OBERON
```

Comment:
Why do we need to use SW crypto? Why can't we use CRACEN? Does ED25519 require something that CRACEN cannot process?

Reply:
No, that is mistake.